docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -12,13 +12,16 @@ and [silverstripe/recipe-cms](https://github.com/silverstripe/recipe-cms)).
 
 ## Release Planning
 
-Releases are planned by assigning issues to GitHub milestones,
-according to our [roadmap](http://silverstripe.org/roadmap).
-SilverStripe is split up into many modules listed on [github.com/silverstripe](https://github.com/silverstripe).
+The Silverstripe CMS core team uses an Agile methodology. We aim to keep the latest development branches of each 
+module in a "potentially releasable" state. Releases are planned around an issue backlog according to our 
+[roadmap](https://www.silverstripe.org/software/roadmap/). We operate in 2-week sprints and 
```

Comment:
2-week sprint is rather a part of our internal processes and may change without any warnings or obligations. I don't think we have to mention that in here. Also, if we change to kanban, we won't remember to patch this documentation, so I reckon we should avoid mentioning it here in the first place.

I also believe this might be rather confusing for community since they don't have any ways to see what's in sprints.

Reply:
This kind of fits with my side idea of publishing our change log every two weeks.

I included this part in here just so people would have a generic idea of how we decide our ongoing priorities. I don't think this commits us to always doing it that way ... the previous milestone bit has been wrong for several years as far as I can tell and I never heard any complaints.

We can make it more high level if you want. Maybe something like this.
> The Silverstripe CMS core team uses an Agile methodology. We aim to keep the latest development branches of each module in a "potentially releasable" state. Releases are planned around an issue backlog according to our [roadmap](https://www.silverstripe.org/software/roadmap/). We re-revaluate our priorities every few weeks.